schema/adminschema/v1.json

```diff
@@ -82,8 +82,15 @@
       "ClientRegistrationResponse": {
           "id": "ClientRegistrationResponse",
           "type": "object",
-          "description": "This object is a client registration respones as defined by the OpenID Connect dynamic registration spec. For brevity field names and types have been omitted.",
-          "properties": {}
+          "description": "This object is a client registration respones as defined by the OpenID Connect dynamic registration spec. For brevity responses other than client id and secret have been omitted.",
```

Comment:
this is not really the OpenID Connect dynamic registration anymore (though it closely resembles it) so let's drop all this wording.